src/main.js

```diff
@@ -217,7 +219,17 @@ const logger = {
             info: (...params) => infoLog(module, ...params),
             debug: (...params) => debugLog(module, ...params),
         };
+    },
+
+    setShouldObfuscate: (obf) => {
+        return logger
     }
 };
 
-module.exports = logger;
\ No newline at end of file
+const obfuscator = (content, after = [], keepList = []) => {
+    console.log(shouldObfuscate)
```

Comment:
Should here?

Reply:
whaaat, did a little mess in the previous commit. Fixing

Reply:
done